featuresScore.java

```diff
@@ -0,0 +1,76 @@
+
+public class featuresScore {
```

Comment:
I prefer class names to be PascalCased (the F of features is also capitalized). Unless the rest is okay with camelCase....?